mpc-core/src/protocols/rep3.rs

```diff
@@ -463,6 +532,14 @@ impl<F: PrimeField, N: Rep3Network> PrimeFieldMpcProtocol<F> for Rep3Protocol<F,
         dst.b[dst_offset..dst_offset + len].clone_from_slice(&src.b[src_offset..src_offset + len]);
     }
 
+    fn print_share(&self, to_print: &Self::FieldShare) {
+        if to_print.a.is_zero() {
+            println!("0")
+        } else {
+            println!("{}", to_print.a)
+        }
+    }
+
```

Comment:
I think the consensus with @0xThemis was that we should just remove this alltogether and maybe introduce a debug_print method that actually opens then share before printing it.

Reply:
done (with a dangerous feature flag)